index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio Mobile Version Skeleton</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <div class="logo">
+            
+           <h4 class="logo">MY Logo</h4>
+            
+        </div>
+        <div class="navbar">
+            <img src="Union.svg" alt="Navbar-Image" class="nav-image">
+        </div>
+    </header>
+
+    <section class="headling">
+        <h2 class="intro">
+            I’m Mariane Glad to see you!
+        </h2>
+
+        <p class="professional-details">
+            I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+
+        <h4 class="social-media-intro">LET’S CONNECT</h4>
+
+        <img src="social-media-icons.svg" alt="" class="social-icon">
+
+        <img src="header-shapes-mobile.svg" alt="" class="header-shape-image">
+        <!-- <img src="header-shapes mobile@2x.svg" alt="" class="header-background"> -->
+    </section>
+    
+
+
+</body>
+</html>
```

Comment:
- [ ] Kindly match the spacing around this section to be more like the Figma design.
- [ ] Please fix the image to be in the background instead of using an `img` tag.

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/62290545/164246426-b8492039-8af5-482d-b515-6ff5bd8ed41d.png) | ![image](https://user-images.githubusercontent.com/62290545/164246596-a74dddb5-526f-4c77-a315-73dfa08e12d2.png) |